utilities.py

```diff
@@ -42,6 +43,10 @@
 plt.rc('font', **font)
 plt.rc('legend', fancybox = True, framealpha=1, shadow=True, borderpad=1)
 
+# Import state geographical data
+states = geopandas.read_file("https://raw.githubusercontent.com/edgi-govdata-archiving/ECHO-Geo/main/cb_2018_us_state_500k.json")
```

Comment:
We can actually pull this from the Stony Brook University server, instead of Github, using other utility functions in development, but for now this is great!!!